src/utils/utils/sorting.ts

```diff
@@ -0,0 +1,16 @@
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+const getValueByPath = (obj: K8sResourceCommon, path: string) => {
+  const pathArray = path?.split('.');
+  return pathArray?.reduce((acc, field) => acc?.[field], obj);
+};
+
+export const sorting = (path: string, direction: string) => (a, b) => {
```

Comment:
This needs a more descriptive name and it should be phrased as a verb. `sortIngressesByHost` below is a great example.

Reply:
is `sortingCommonColumnsByPath` okay?